src/convertToObject.js

```diff
@@ -6,7 +6,31 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // if the sourceString is empty return an empty object
+  if (!sourceString) {
+    return {};
+  }
+
+  // remove all tab and new line characters
+  // replaceAll doesn't work on this version of Node
+  const clearedString = sourceString.replace(/[\t\n]/g, '');
+  const stylesArr = clearedString.split(';');
```

Comment:
[CODE STYLE] - The variable name `stylesArr` is not very descriptive and could be improved to enhance readability. Consider naming it in a way that reflects its contents, such as `styleDeclarations` or `stylePairs`.